### DISCUSSION TITLE
output for v10 and u10

### DISCUSSION THREAD

**Discussion by pgoddard08** [E3SM-Project/E3SM#6800] [E3SM model help]
Hi,
I would like to ensure the atmo model output zonal and meridional wind at 10 m. Are there variables names I can include in the user_nl_eam file?

**Comment by rljacob** [comment]
There is a "U10" available but, oddly, no "V10".

**Reply by pgoddard08** [reply]
Thank you Robert - Just verifying that U10 is zonal 10 m winds, not capital "U"10 which might mean 10 m wind speed?

**Reply by chengzhuzhang** [reply]
I understand that U10 is the 10m wind speed, not the zonal component of wind in EAM. It is possible to add output for UBOT, and VBOT (bottom level winds) in user_nl_eam, but I don't know if it is possible to directly output zonal and meridional wind at 10m.
